api/templates/military-selective.xml

```diff
@@ -9,6 +9,12 @@
     <LegalExemptionExplanation>{{textarea .props.Explanation}}</LegalExemptionExplanation>
     <LegalExemptionExplanationComment></LegalExemptionExplanationComment>
     <NumberComment></NumberComment>
+    <DNKExplanation></DNKExplanation>
```

Comment:
Did you mean to add lines 12 and 13? Looks like duplicates of 18 and 19. If so do we need to fill in the explanation?

Reply:
Admittedly, this is where my lack of knowledge on how the XML is expected to look downstream comes into play. It looks like in a lot of areas we have blank XML tags in the template, like `<LegalExemptionExplanationComment></LegalExemptionExplanationComment>`, so for consistency's sake I return the empty tags in both conditions, but I'm not totally sure that is necessary. The way I have it now returns the XML exactly as it was before, the only difference is in one case the explanation is in the `LegalExemptionExplanation` tag, and in the other the `DNKExplanation`. Open to change this around if you think we can omit the empty tags.

Reply:
Ah I think I follow the code now. Sorry for the confusion it was on my end.